scripts/build-local.js

```diff
@@ -0,0 +1,30 @@
+var path = require('path')
+var fs = require('fs')
+var spawn = require('child_process').spawn
+
+// Runs `npm run build-local` if the lib dir is absent
+
+fs.access(path.join(__dirname, '..', 'lib'), function(err) {
+  if (err) {
+    // If we're installed as a dependency in another project, install our own node_modules,
+    // which are required to run build-local
+    fs.access(path.join(__dirname, '..', 'node_modules', 'babel-plugin-external-helpers'), function(err) {
```

Comment:
I think this check can be removed and we instead always run `npm install`

Reply:
No, it needs to be there, or we'll end up with infinite recursion.

After the script runs `npm install`, the `postinstall` script runs again, and this time we *don't* want to run `npm install`.